timeline/rules.py

```diff
@@ -44,3 +44,9 @@
 
 # Allow viewing classified site-wide event
 rules.add_perm('timeline.view_classified_site_event', rules.is_superuser)
+
+# Allow viewing extra data of event
```

Comment:
Just a small note, maybe make it clear in this comment that this perm is specifically for viewing project events.